core/tx_pool.go

```diff
@@ -860,6 +863,13 @@ func (pool *TxPool) addTx(tx *types.Transaction, local bool) error {
 	// Try to inject the transaction and update any state
 	replace, err := pool.add(tx, local)
 	if err != nil {
+		pool.txnErrorSink([]types.RPCTransactionError{
+			{
```

Comment:
did gofmt accept this?

Reply:
Yep, passed Travis and I just ran go fmt on it.